packages/ui/src/components/ExpandableCard/__stories__/Header.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import type { StoryFn } from '@storybook/react'
+import { ProductIcon } from '@ultraviolet/icons'
+import { ExpandableCard } from '..'
+import { Button, MenuV2, Stack, Text } from '../..'
+
+export const Header: StoryFn<typeof ExpandableCard> = () => (
```

Comment:
Can you add a bigger height on this story so we can see the content of the menu?

<img width="1046" alt="Screenshot 2024-07-10 at 12 05 14" src="https://github.com/scaleway/ultraviolet/assets/15812968/728677ee-74b9-40fd-b196-6e3577815527">